updater/lib/dependabot/base_command.rb

```diff
@@ -8,21 +8,28 @@
 require "dependabot/logger/formats"
 require "dependabot/environment"
 
+require "sorbet-runtime"
+
 module Dependabot
   class RunFailure < StandardError; end
 
   class BaseCommand
+    extend T::Sig
+
     # Implement in subclass
+    sig { overridable.void }
     def perform_job
       raise NotImplementedError
```

Comment:
Can you remove the implementation here? Same for all the methods below that just `raise NotImplementedError`.